taskt/Core/Automation/Commands/Dictionary Commands/AddDictionaryItemCommand.cs

```diff
@@ -1,91 +1,71 @@
-        [Attributes.PropertyAttributes.SampleUsage("vMyDictionary")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Dictionary Name")]
+        [InputSpecification("Select the dictionary variable to add item to.")]
+        [SampleUsage("{vMyDictionary}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_DictionaryName { get; set; }
 
         [XmlElement]
-        [Attributes.PropertyAttributes.PropertyDescription("Define Keys and Values")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the Keys and Values required for your dictionary")]
-        [Attributes.PropertyAttributes.SampleUsage("")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Keys and Values")]
+        [InputSpecification("Enter Keys and Values required for the dictionary.")]
+        [SampleUsage("")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public DataTable v_ColumnNameDataTable { get; set; }
 
-
         public AddDictionaryItemCommand()
         {
-            this.CommandName = "AddDictionaryItemCommand";
-            this.SelectionName = "Add Dictionary Item";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "AddDictionaryItemCommand";
+            SelectionName = "Add Dictionary Item";
+            CommandEnabled = true;
+            CustomRendering = true;
 
             //initialize Datatable
-            this.v_ColumnNameDataTable = new System.Data.DataTable
+            v_ColumnNameDataTable = new DataTable
             {
                 TableName = "ColumnNamesDataTable" + DateTime.Now.ToString("MMddyy.hhmmss")
             };
 
-            this.v_ColumnNameDataTable.Columns.Add("Keys");
-            this.v_ColumnNameDataTable.Columns.Add("Values");
-
-
-
+            v_ColumnNameDataTable.Columns.Add("Keys");
+            v_ColumnNameDataTable.Columns.Add("Values");
         }
 
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
-            var dictionaryName = v_DictionaryName.ConvertToUserVariable(sender);
+            var engine = (AutomationEngineInstance)sender;
             var dataSetVariable = LookupVariable(engine);
 
-
             Dictionary<string, string> outputDictionary = (Dictionary<string, string>)dataSetVariable.VariableValue;
 
             foreach (DataRow rwColumnName in v_ColumnNameDataTable.Rows)
             {
                 outputDictionary.Add(rwColumnName.Field<string>("Keys"), rwColumnName.Field<string>("Values"));
```

Comment:
both fields need a .ConvertToUserVariable() at the end of them

Reply:
done